src/Page/Board.elm

```diff
@@ -288,11 +292,32 @@ notesView notesId =
         []
 
 
-subtasksView : List ( String, TaskItem ) -> Html Msg
-subtasksView subtasks =
+subtasksView : GlobalSettings -> List ( String, TaskItem ) -> Html Msg
+subtasksView globalSettings subtasks =
+    let
+        maybeFilterList list = 
+            if globalSettings.hideCompletedSubtasks then
```

Comment:
Just noticed a small nitpick as I was typing up the PR, so typing up a comment as I'm done coding for the night.

There is a case mismatch between `hideCompletedSubtasks` and `subTaskDisplayLimit`. The `t` in subtask is lower in one and upper in the other. Should make this consistent.